partner_multi_company/models/res_partner.py

```diff
@@ -48,7 +49,10 @@ def create(self, vals):
         # We must suspend security during this creation because it fails due to
         # all the mail stuff in between that confuses security rules
         obj = getattr(self, 'suspend_security', lambda: self)()
-        return super(ResPartner, obj).create(vals)
+        new_record = super(ResPartner, obj).create(vals)
+        # Call check_access_rules on new record to honor security rules
+        self.browse(new_record.id).check_access_rule('create')
```

Comment:
I don't see this needed

Reply:
You're right this belong to other pourpose